merge_data.py

```diff
@@ -110,7 +110,7 @@ def main():
                 source_usmart = pd.concat(
                     [
                         source_usmart,
-                        pd.read_csv(
+                        pd.read_json(
                             folder + r"/" + filename,
                             parse_dates=["DateCreated", "DateUpdated"],
```

Comment:
This line caused an error for me. I think it needs to be `convert_dates` instead